src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -57,10 +67,19 @@ export function ConfigEditor(props: Props) {
     const res: Secret = await getBackendSrv().post(resourcesURL + '/secret', { secretARN: arn });
     return res;
   };
+  const fetchClusters = async () => {
+    const res: Cluster[] = await getBackendSrv().get(resourcesURL + '/clusters');
+    return res.map((c) => ({
+      label: c.clusterIdentifier,
+      value: c.clusterIdentifier,
+      description: `${c.endpoint.address}:${c.endpoint.port}/${c.database}`,
```

Comment:
this returns a database, should we make the input disable and fill automatically based on clusterId selected?

Reply:
I don't think we should, that's the default database, it's possible to create a different one in the same cluster and use that one instead. I would also not display this URL in this selector as the description for the same reason.

Reply:
Thanks!
So I should use the database entered in the db field to create the url in the datasource card instead of this default one?

Reply:
yes, that would make sense